cmd/fyne_settings/settings/appearance.go

```diff
@@ -10,20 +12,36 @@ import (
 	"fyne.io/fyne"
 	"fyne.io/fyne/app"
 	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/internal/painter"
 	"fyne.io/fyne/layout"
 	"fyne.io/fyne/theme"
+	"fyne.io/fyne/tools/playground"
 	"fyne.io/fyne/widget"
 )
 
 const (
 	systemThemeName = "system default"
 )
 
+var (
+	colorNames = []string{theme.ColorBlue, theme.ColorGreen, theme.ColorYellow,
+		theme.ColorOrange, theme.ColorRed, theme.ColorGray}
+	colorValues = map[string]color.Color{
```

Comment:
Add Indigo, Violet/Purple for a full rainbow.
Brown would be a nice addition too.

Reply:
let's not add to this by covering greyscale as well - this is only adding colours that can work as primary and you need to get the shade just right to do that so most greys won't work

Reply:
Added briwn and purple too, indigo seemed like it didn't match the overall aim to match [saturation]?

Reply:
This is fine for here, but there is a problem using this in colour picker as it has a row of primary colours and a row of greyscale - it will be odd to have grey in the primary colours above the row of greyscale

Reply:
You can remove Color.Grey from the list of colours you use surely? It's not an all-or-nothing API

Reply:
ok